queries.yaml

```diff
@@ -1841,6 +1841,38 @@ post_process:
       # need to be a large tolerance.
       simplify_tolerance: 1.0
 
+  # drop labels on roads which are too short to render.
+  - fn: vectordatasource.transform.drop_names_on_short_boundaries
+    params:
+      source_layer: roads
+      start_zoom: 8
```

Comment:
At zooms 10, 11, 12 this is only dropping a few road names because by kind and kind_detail we already drop many of the names at the earlier zooms.

So it's a build time and $ question if we want to do this for zoom 10, 11, 12 or just at 13 (nominal zoom 14). I think it's fine to keep the `start_zoom: 8` for now and we can modify to taste later.